config/manifests/gateway-api-inference-extension/values.yaml

```diff
@@ -0,0 +1,13 @@
+inferenceExtension:
+  replicas: 1
+  image:
+    name: epp
+    hub: us-central1-docker.pkg.dev/k8s-staging-images/gateway-api-inference-extension
+    tag: main
+    pullPolicy: Always
+
+  name: inference-gateway-ext-proc
```

Comment:
This should probably have the name of the inferencePool in it by default. So if the inference pools is called `base`, maybe this is called `base-epp`